fluxfw-gcp/tf/examples/notebooks/README.md

```diff
@@ -0,0 +1,171 @@
+# Flux Framework Notebook Example
+
+This example demostrates how to deploy a [Jupyter notebook](https://jupyter.org/try-jupyter/lab/) in a flux allocation on GCP. Several
+example notebooks are provided that illustrate using the [flux-framework Python API](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/python/index.html) to interact
+with flux programatically in a Jupyter context.
+
+## Deployment
+
+Initialize the deployment with the command:
+
+```bash
+terraform init
+```
+
+Make a copy of the `notebooks.tfvars.example` file:
+
+```bash
+cp notebooks.tfvars.example notebooks.tfvars
+```
+
+Modify the notebooks.tfvars to specify the desired configuration. For example you may want more than
+one login node, you probably want more than one compute node, you may want a different machine_type, 
+and/or you may want to add another node pool, etc.
+
+Note that many of the module variables receive their values from the remote state of other components.
+
+Deploy the cluster with the command:
+
+```bash
+terraform apply -var-file notebooks.tfvars
+```
+
+## Setup
+
+### Node Allocation
+
+Once the `terraform apply` command completes you can log into your cluster using the command:
+
+```bash
+gcloud compute ssh nb-login-001
+```
+
+Check the state of the cluster with the command:
+
+```bash
+flux resource list
+```
+
+When the cluster is ready you will see output that looks like:
+
+```bash
+     STATE PROPERTIES NNODES   NCORES    NGPUS NODELIST
+      free x86-64,e2       1        2        0 nb-login-001
+      free t2a,arm64       8      128        0 nb-compute-a-[001-008]
+ allocated                 0        0        0
+      down                 0        0        0
+```
+
+Now you can create an allocation for your work using the command:
+
+```bash
+flux mini alloc -N4 --requires=arm64
+```
+
+Flux will allocate four nodes and give you a new shell on one of them.
+
+### Jupyter Installation
+
```

Comment:
Could the conda install be part of the startup?

Reply:
It could but this way puts it in the user's home directory, which may be preserved across cluster instances since the filestore instance isn't torn down with the cluster, so that it only has to be done once (per filestore instantiation) and isn't necessary on all the compute nodes.

Reply:
It also doesn't run afoul of the license prompt when you install conda.